docker/build_standalone-tcks.sh

```diff
@@ -149,7 +149,7 @@ for tck in ${TCK_LIST[@]}; do
     JAXWS_SPECIFIC_PROPS=""
   elif [ "websocket" == "$tck" ]
   then
-    TCK_SPECIFIC_PROPS="-Dwebsocket.classes=$JAKARTA_JARS/modules/jakarta.websocket-api.jar:$JAKARTA_JARS/modules/jakarta.servlet-api.jar:$JAKARTA_JARS/modules/jakarta.inject-api.jar:$JAKARTA_JARS/modules/jakarta.enterprise.cdi-api.jar:$JAKARTA_JARS/modules/jakarta.activation-api.jar:$JAKARTA_JARS/modules/glassfish-corba-omgapi.jar"
+    TCK_SPECIFIC_PROPS="-Dwebsocket.classes=$JAKARTA_JARS/modules/jakarta.websocket-client-api.jar:$JAKARTA_JARS/modules/jakarta.websocket-api.jar:$JAKARTA_JARS/modules/jakarta.servlet-api.jar:$JAKARTA_JARS/modules/jakarta.inject-api.jar:$JAKARTA_JARS/modules/jakarta.enterprise.cdi-api.jar:$JAKARTA_JARS/modules/jakarta.activation-api.jar:$JAKARTA_JARS/modules/glassfish-corba-omgapi.jar"
```

Comment:
Please add newly introduced jars for download here - https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/docker/pom.xml#L37, so that download logic (https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/docker/build_jakartaeetck.sh#L98) picks up the jars.

Reply:
Does the build not follow the dependencies? `jakarta.websocket-api` depends on `jakarta.websocket-client-api`

Reply:
As per below logs from PR branch, dependencies are not followed: 
```
mvn -f /home/guru/oracle/temp/jakartaee-tck/docker/pom.xml -Pstaging dependency:copy-dependencies -DoutputDirectory=/home/guru/oracle/temp/jakartaee-tck/modules -Dmdep.stripVersion=true
[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------------< jakarta:dependency_copy >-----------------------
[INFO] Building dependency_copy 1
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.10:copy-dependencies (default-cli) @ dependency_copy ---
[INFO] Copying asm-util-9.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/asm-util.jar
[INFO] Copying glassfish-corba-internal-api-4.2.4.jar to /home/guru/oracle/temp/jakartaee-tck/modules/glassfish-corba-internal-api.jar
[INFO] Copying mimepull-1.9.13.jar to /home/guru/oracle/temp/jakartaee-tck/modules/mimepull.jar
[INFO] Copying jakarta.faces-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.faces-api.jar
[INFO] Copying jakarta.batch-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.batch-api.jar
[INFO] Copying asm-analysis-9.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/asm-analysis.jar
[INFO] Copying webservices-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/webservices-api.jar
[INFO] Copying jakarta.xml.ws-api-4.0.0-RC1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.xml.ws-api.jar
[INFO] Copying webservices-osgi-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/webservices-osgi.jar
[INFO] Copying pfl-dynamic-4.1.2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/pfl-dynamic.jar
[INFO] Copying webservices-tools-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/webservices-tools.jar
[INFO] Copying webservices-rt-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/webservices-rt.jar
[INFO] Copying jakarta.servlet.jsp.jstl-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.servlet.jsp.jstl-api.jar
[INFO] Copying jakarta.authentication-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.authentication-api.jar
[INFO] Copying jakarta.xml.soap-api-3.0.0-RC1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.xml.soap-api.jar
[INFO] Copying istack-commons-runtime-3.0.8.jar to /home/guru/oracle/temp/jakartaee-tck/modules/istack-commons-runtime.jar
[INFO] Copying jakarta.enterprise.concurrent-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.enterprise.concurrent-api.jar
[INFO] Copying jakarta.json-api-2.1.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.json-api.jar
[INFO] Copying jakarta.enterprise.cdi-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.enterprise.cdi-api.jar
[INFO] Copying woodstox-core-6.2.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/woodstox-core.jar
[INFO] Copying slf4j-api-1.7.30.jar to /home/guru/oracle/temp/jakartaee-tck/modules/slf4j-api.jar
[INFO] Copying jakarta.json.bind-api-3.0.0-SNAPSHOT.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.json.bind-api.jar
[INFO] Copying glassfish-corba-csiv2-idl-4.2.4.jar to /home/guru/oracle/temp/jakartaee-tck/modules/glassfish-corba-csiv2-idl.jar
[INFO] Copying jaxb-runtime-2.3.2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jaxb-runtime.jar
[INFO] Copying xmlsec-2.1.5.jar to /home/guru/oracle/temp/jakartaee-tck/modules/xmlsec.jar
[INFO] Copying jakarta.security.enterprise-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.security.enterprise-api.jar
[INFO] Copying jakarta.validation-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.validation-api.jar
[INFO] Copying jakarta.transaction-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.transaction-api.jar
[INFO] Copying jakarta.activation-api-2.1.0-RC2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.activation-api.jar
[INFO] Copying jakarta.xml.bind-api-4.0.0-RC2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.xml.bind-api.jar
[INFO] Copying pfl-tf-4.1.2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/pfl-tf.jar
[INFO] Copying jakarta.interceptor-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.interceptor-api.jar
[INFO] Copying resolver-20050927.jar to /home/guru/oracle/temp/jakartaee-tck/modules/resolver.jar
[INFO] Copying jakarta.resource-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.resource-api.jar
[INFO] Copying glassfish-corba-omgapi-4.2.4.jar to /home/guru/oracle/temp/jakartaee-tck/modules/glassfish-corba-omgapi.jar
[INFO] Copying pfl-asm-4.0.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/pfl-asm.jar
[INFO] Copying jakarta.mail-api-2.1.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.mail-api.jar
[INFO] Copying asm-9.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/asm.jar
[INFO] Copying jakarta.authorization-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.authorization-api.jar
[INFO] Copying stax2-api-4.2.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/stax2-api.jar
[INFO] Copying pfl-basic-tools-4.0.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/pfl-basic-tools.jar
[INFO] Copying glassfish-corba-orb-4.2.4.jar to /home/guru/oracle/temp/jakartaee-tck/modules/glassfish-corba-orb.jar
[INFO] Copying jakarta.activation-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.activation.jar
[INFO] Copying org.osgi.core-6.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/org.osgi.core.jar
[INFO] Copying exception-annotation-processor-4.2.4.jar to /home/guru/oracle/temp/jakartaee-tck/modules/exception-annotation-processor.jar
[INFO] Copying jakarta.ws.rs-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.ws.rs-api.jar
[INFO] Copying FastInfoset-1.2.16.jar to /home/guru/oracle/temp/jakartaee-tck/modules/FastInfoset.jar
[INFO] Copying management-api-3.2.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/management-api.jar
[INFO] Copying jakarta.servlet.jsp-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.servlet.jsp-api.jar
[INFO] Copying txw2-2.3.2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/txw2.jar
[INFO] Copying stax-ex-1.8.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/stax-ex.jar
[INFO] Copying jakarta.jms-api-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.jms-api.jar
[INFO] Copying jakarta.persistence-api-3.1.0-RC1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.persistence-api.jar
[INFO] Copying webservices-api-osgi-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/webservices-api-osgi.jar
[INFO] Copying commons-codec-1.13.jar to /home/guru/oracle/temp/jakartaee-tck/modules/commons-codec.jar
[INFO] Copying jakarta.websocket-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.websocket-api.jar
[INFO] Copying asm-tree-9.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/asm-tree.jar
[INFO] Copying jakarta.annotation-api-2.1.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.annotation-api.jar
[INFO] Copying gmbal-4.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/gmbal.jar
[INFO] Copying pfl-tf-tools-4.0.1.jar to /home/guru/oracle/temp/jakartaee-tck/modules/pfl-tf-tools.jar
[INFO] Copying jakarta.ejb-api-4.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.ejb-api.jar
[INFO] Copying webservices-extra-jdk-packages-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/webservices-extra-jdk-packages.jar
[INFO] Copying pfl-basic-4.1.2.jar to /home/guru/oracle/temp/jakartaee-tck/modules/pfl-basic.jar
[INFO] Copying jakarta.el-api-5.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.el-api.jar
[INFO] Copying jakarta.inject-api-2.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.inject-api.jar
[INFO] Copying jakarta.servlet-api-6.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jakarta.servlet-api.jar
[INFO] Copying jaxb-osgi-3.0.0.jar to /home/guru/oracle/temp/jakartaee-tck/modules/jaxb-osgi.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

```

Reply:
Ack. I'll work on a fix.

Reply:
Fixed.